packages/lesswrong/lib/collections/posts/helpers.ts

```diff
@@ -10,7 +10,7 @@ import moment from '../../moment-timezone';
 import { max } from "underscore";
 import { TupleSet, UnionOf } from '../../utils/typeGuardUtils';
 
-export const postCategories = new TupleSet(['post', 'linkpost', 'question'] as const);
+export const postCategories = new TupleSet(['post', 'linkpost', 'question', 'dialogue'] as const);
```

Comment:
I thought this change was useful, but I think upon reflection it wasn't. Planning to revert.